pkg/services/librarypanels/api.go

```diff
@@ -49,3 +51,28 @@ func (lps *LibraryPanelService) deleteHandler(c *models.ReqContext) api.Response
 
 	return api.Success("Library panel deleted")
 }
+
+// getHandler handles GET /api/library-panels/:panelId
+func (lps *LibraryPanelService) getHandler(c *models.ReqContext) api.Response {
+	libraryPanel, err := lps.getLibraryPanel(c, c.ParamsInt64(":panelId"))
+
+	if err != nil {
```

Comment:
I would recommend not having a blank line between an error check and its origin, since it makes verifying harder for someone reading the code (as the connection is no longer obvious):
```suggestion
	libraryPanel, err := lps.getLibraryPanel(c, c.ParamsInt64(":panelId"))
	if err != nil {
```

Reply:
Sure I agree, will fix this in my next commit.

Reply:
Fixed